### PR TITLE
new process idea

### PR DESCRIPTION
**Pull request by clouserw**
New process idea that we talked about a little bit last sprint and a little be this sprint.  I think this is what folks were talking about?

Along with this, we'd drop the Monday planning meeting.

### CODE REVIEW

**Comment by jaredhirsch** [comment]
:+1: lgtm, I'm excited to try this out!

File: docs/process.md [hunk 1]
```diff
@@ -2,59 +2,83 @@
...
 
-## Current Milestone
+* *Untriaged*:  These are new issues we haven't looked at yet.
+* *Backlog*:  We've looked at these issues, but there are still some open
+  questions or were not ready to start working on them yet.
+* *Ready*:  These issues have been triaged, questions are answered, assets are
+  available, and work is ready to begin.  To be in this column an issue *must*
+  have a `size` label.
+* *In Progress*:  These issues are being actively worked on.  To be in this
+  column an issue must be assigned to someone.  PRs also land here.
```
Comment (ckprice) [inline docs/process.md]: We could use an `In Review` column here.

File: docs/process.md [hunk 2]
```diff
@@ -2,59 +2,83 @@
...
+* Ask the question in the issue and @-tag the person who can help
+* Use the `needs:discussion`, `needs:moreinfo`, `needs:product`, `needs:qa`,
+  and `needs:ux` labels
+* Move the issue to *Backlog*
 
-This is linked to the `Ready` column on our waffle board.
+If the issue is ready to be worked on:
+* Assign an appropriate `size` label
+* Consider any other labels (perhaps it's a `good-first-bug`?)
+* Move the issue to *Ready*
```
Comment (ckprice) [inline docs/process.md]: Adding a bit more detail to the 3 working columns as a recommendation.

All of our meetings will become (20 minute?) triage meetings with the Waffle board on a shared screen. Going over each column and each issue.

1. `In Review` these are PR's and they should already be assigned.
2. `In Progress` these should all either be assigned, or ready to move over to `In Review`. Ideally someone will assign themselves for review _in meeting_.
3. `Ready` things in this column are primed and ready to go. We will move these over _in meeting_ with an assignee.

File: docs/process.md [hunk 3]
```diff
@@ -2,59 +2,83 @@
...
+## Labels
 
-This label is applied to work we have identified in our planning meeting as targeting our current milestone.
+Size labels are roughly defined as:
 
-This is linked to the `Current Milestone` column on our waffle board.
+- `size:small` ~1 hour
+- `size:medium` ~1 day
+- `size:large` ~1 week
+- `size:too large` >1 week
```
Comment (ckprice) [inline docs/process.md]: I don't know how important/necessary sizing becomes in Kanban. Given we aren't loading a finite amount of time.

**Pull request merged**